html/changelogs/implant_eggs_larvae_scans.yml

```
@@ -0,0 +1,7 @@
+author: Vrow
+
+delete-after: True
+
+changes:
+  - bugfix: "Fixes medscans/body scanners/operating consoles thinking there's something else when someone only has an implant in them."
+  - bugfix: "Fixes the above counting each and every single greimorian larva inside a limb that has eggs implanted."
```

```suggestion
  - bugfix: "Fixes the body scanner subsystem from counting each and every single greimorian larva inside a limb that has eggs implanted."
```
I don't recommend using works like 'the above' for your changelog, be precise for what changed.

---

No offense, but you say "be precise" and then generalize literally every device that uses the medical scan format into "body scanner subsystem"

---

There is no need to be picky about wording if the changelog is clear in purpose and all changes are documented. Suggest author resolve this conversation and the above